File: Sopra/finished_project/sopra-fs19-group03-server/src/main/java/ch/uzh/ifi/seal/soprafs19/dto/incoming/ReadyStateMessage.java
package ch.uzh.ifi.seal.soprafs19.dto.incoming;


public class ReadyStateMessage implements Message {
    private Long userId;
    private boolean ready;


    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public boolean isReady() {
        return ready;
    }

    public void setReady(boolean ready) {
        this.ready = ready;
    }
}

File: Sopra/finished_project/sopra-fs19-group03-client/src/components/shared/models/outgoingMessages/ReadyStateMessage.js
import "../EnumsJS";

class ReadyStateMessage {
    constructor(userId, ready){
        this.userId = userId;
        this.ready = ready;
    }
    getUserId(){return this.userId;}
    getReady(){return this.ready;}
    setUserId(userId){this.userId = userId;}
    setReady(ready){this.ready = ready;}
}
export default ReadyStateMessage;